docs/user-guide/deploy-chart/chart-group.md

```diff
@@ -7,39 +7,35 @@ Create Chart Group And Deploy App store applications.
 
 ## 1. Create Group 
 
-Select the `Charts` section from the left pane, you will be landed to the `Chart Store` page. 
-Click on `Discover` and Click Button on right top `Create Group`.
+1. Select `Charts` from the left pane, you will be landed to the `Chart Store` page. 
+2. Click on `Create Group` button present in the upper right corner.
 
-![](../../.gitbook/assets/chart-group-1.png)
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploy-chart/chart-group/chart-group-1.jpg)
 
-It will pop-up a dialog, fill required data.
+3. It will pop-up a dialog box where you can enter `name` and `description`(optional) for the chart-group and then click on `Create Group`.
 
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploy-chart/chart-group/chart-group-2.jpg)
 
-And click on right bottom button `Create Group`.
-![](../../.gitbook/assets/chart-group-2.png)
-
-It will land you to the page where you can discover charts to add in this group and later deploy by single click.
+4. This will redirect you to the page where you can discover or search charts to add in the chart-group and later deploy by single click.
 
 ## 2. Add Charts To Group 
 
+1. Choose any chart and add by `+` sign and click on `save` to save the chart group.
 
-![](../../.gitbook/assets/chart-group-3.png)
-
-Choose any chart and add by plus button, save this group by pressing `Save`.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploy-chart/chart-group/chart-group-3.jpg)
 
-For Running Instances and Group Detail Click on `Group Detail` from here you can Edit yous charts as well.
+2. Click on `Group Detail` to see all the running instances and group details. You also can edit the chart group from here.
```

Comment:
You can also edit the chart group from here. Instead of You also can sounds more better in my opinion.

Reply:
done